modules/virt-vm-storage-volume-types.adoc

```diff
@@ -30,6 +30,10 @@ PVC.
 DataVolumes build on the *persistentVolumeClaim* disk type by managing the process
 of preparing the virtual machine disk via an import, clone, or upload operation.
 VMs that use this volume type are guaranteed not to start until the volume is ready.
++
+Define DataVolumes with a `type` of *dataVolume*. If another volume type is
```

Comment:
`dataVolume` and `PersistentVolumeClaim` have the wrong syntax highlighting. Only UI elements and text that is bolded for emphasis should be enclosed in asterisks per [the 4.x markup reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Reply:
Fixed, added ticks and removed asterisks.